openshift/sysdig-team/team.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: ops.gov.bc.ca/v1alpha1
+kind: SysdigTeam
+metadata:
+  name: devops-sso-prod-sysdigteam
+spec:
+  team: 
+    description: The Sysdig Team for the SSO Prod Namespaces
+    users:
+    - name: ConradBoydElliottGustafson
```

Comment:
it's not github account but email address

Reply:
oh but which email address? their github id's registered email addres?